assets/tests/components/vehicleCard.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import "@testing-library/jest-dom"
+import { Vehicle } from "../../src/realtime"
+import { Route } from "../../src/schedule"
+import * as dateTime from "../../src/util/dateTime"
+import vehicleFactory from "../factories/vehicle"
+import routeFactory from "../factories/route"
+
+import userEvent from "@testing-library/user-event"
+import VehicleCard from "../../src/components/vehicleCard"
+import { useNearestIntersection } from "../../src/hooks/useNearestIntersection"
+import { RoutesProvider } from "../../src/contexts/routesContext"
+jest
+  .spyOn(dateTime, "now")
+  .mockImplementation(() => new Date("2018-08-15T17:41:21.000Z"))
+
```

Comment:
question(non-blocking): any particular reason for this timestamp?

I've been idly mulling over if switching to a unix timestamp would be better for testing (eg let's findout what skate does when it's back in 0 seconds, 1970)
but I think this is mostly bikeshedding.

Reply:
Great question! I pulled this mock from the search results test file, but turns out it is not needed for this component. The mock is only needed if we were to test that the operator logon is shown in this card (the default last logon time defined in the vehicle factory is [2018-08-15T13:38:21.000Z](https://github.com/mbta/skate/blob/15df510fc794a8b9a8e5c4e3045b46e33aed322a/assets/tests/factories/vehicle.ts#L19), but since that data is not present in this new card (at least for now) the mock can be omitted.